api/v1alpha1/hostedcluster_types.go

```diff
@@ -24,6 +24,12 @@ const (
 	//KonnectivityAgentImageAnnotation is a temporary annotation that allows the specification of the konnectivity agent image.
 	//This will be removed when Konnectivity is added to the Openshift release payload
 	KonnectivityAgentImageAnnotation = "hypershift.openshift.io/konnectivity-agent-image"
+	//ClusterAPIManagerImage is a temporary annotation that allows the specification of the cluster api manager image.
+	//This will be removed when the proper version of the cluster api manager is added to the Openshift release payload
+	ClusterAPIManagerImage = "hypershift.openshift.io/capi-manager-image"
```

Comment:
This images will be built owned and pulled from our internal pipeline but I'm not sure they will ever be part of the payload, they don't necessarily match an ocp version.

Reply:
aha ok let me adjust the comment

Reply:
comments adjusted